pgvectorscale/src/access_method/mod.rs

```diff
@@ -88,27 +91,23 @@ fn amhandler(_fcinfo: pg_sys::FunctionCallInfo) -> PgBox<pg_sys::IndexAmRoutine>
 // This SQL is made idempotent so that we can use the same script for the installation and the upgrade.
 extension_sql!(
     r#"
-DO $$
-DECLARE
-  c int;
-BEGIN
-    SELECT count(*)
-    INTO c
-    FROM pg_catalog.pg_opclass c
-    WHERE c.opcname = 'vector_cosine_ops'
-    AND c.opcmethod = (SELECT oid FROM pg_catalog.pg_am am  WHERE am.amname = 'diskann');
-
-    IF c = 0 THEN
-        CREATE OPERATOR CLASS vector_cosine_ops DEFAULT
-        FOR TYPE vector USING diskann AS
-	        OPERATOR 1 <=> (vector, vector) FOR ORDER BY float_ops;
-    END IF;
-END;
-$$;
+    DROP OPERATOR CLASS IF EXISTS vector_cosine_ops USING diskann;
```

Comment:
I am doubtful its safe/good to always drop the class like this. I'd rather do it if its missing or missing the function. Also what happens if an index already exists and we try dropping the opclass? Wouldn't the drop fail if no CASCADE? And the CASCADE would drop the index. This seem problematic.